OCR/tests/alignment_test.py

```diff
@@ -15,13 +15,23 @@
 
 class TestAlignment:
     def test_random_warp(self):
-        transformed = RandomPerspectiveTransform(filled_image_path).transform(distortion_scale=0.1)
+        transformed = RandomPerspectiveTransform(filled_image_path).random_transform(distortion_scale=0.1)
         assert np.median(cv.absdiff(np.array(transformed), filled_image)) > 0
 
     def test_alignment_filled(self):
         aligner = ImageHomography(template_image_path)
-        warped_image = np.array(RandomPerspectiveTransform(filled_image_path).transform(distortion_scale=0.1))
+        warped_image = np.array(RandomPerspectiveTransform(filled_image_path).random_transform(distortion_scale=0.1))
         aligned = aligner.transform_homography(warped_image)
         res = cv.absdiff(aligner.template, aligned)
         assert aligner.template.shape == warped_image.shape
         assert np.median(res) == 0
+
+    def test_inverse_alignment(self):
```

Comment:
can you please add some comments for someone who is not super familiar with this code
- a high level description of this test, for example: we are warping an image with a known values, then we estimate the warp, then compare the estimated value to the original known value and asserting that the difference is not too much.
- why the value of 1 was selected
-